Airways/RNAV/Upper/UN160.txt

```diff
@@ -0,0 +1,4 @@
+LIZAD LIZAD NAKID NAKID
+NAKID NAKID LND   LND  
+LND   LND   BOGMI BOGMI
```

Comment:
LND-INSUN-BOGMI-LEDGO

Reply:
@NathanInnes can you change this to the following and commit the changes?

```
LND   LND   INSUN INSUN
INSUN INSUN BOGMI BOGMI
BOGMI BOGMI LEDGO LEDGO
```

Reply:
@NathanInnes  

The full routing begins at LIZAD - please check the documents which we use for our data.

[
Here](https://drive.google.com/open?id=1vc_v8TY0DkBPV-jCRU59Kq3FD29KP_tF)